app/controllers/UserController.php

```diff
@@ -495,30 +495,27 @@ public static function message($method)
 
     public static function reactiveAccount($method){
         if ($method == 'POST') {
-            $email = isset($_POST['email']) ? trim($_POST['email']) : null;
             $codeOTP = isset($_POST['codeOTP']) ? trim($_POST['codeOTP']) : null;
```

Comment:
Use ``filter_input()`` instead

Reply:
Filter input will return false if not found, **if null is required** [FILTER_NULL_ON_FAILURE](https://www.php.net/manual/en/filter.constants.php#constant.filter-null-on-failure) can be used.
```suggestion
            $codeOTP = trim(filter_input(INPUT_POST, 'codeOTP'));
``